staging/publishing/import-restrictions.yaml

```diff
@@ -14,6 +14,7 @@
 
   # the following are temporary and should go away. Think twice (or more) before adding anything here.
   # Main goal: pkg/apis should be as self-contained as possible.
+  - k8s.io/component-helpers/scheduling/corev1
```

Comment:
where do you need this? I would prefer not to add anything here, rather the opposite.

Reply:
Oh wait. These are the imports into k/k/pkg. Please move your line up into the other block of allowed imports. There is nothing bad to import component-helpers from pkg/apis/core.

Also please move the apps/v1 below also into the upper block.

Reply:
Done